### ISSUE TITLE
[Accessbility] Screen readers are not announcing the row headers like (12 am , 1am etc) when we are viewing the events in week view

### ISSUE THREAD

**Issue by sindhuraN** [fullcalendar/fullcalendar#7288]
### Reduced Test Case

https://codepen.io/dcangulo/pen/yrQGVj

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

React

### Bug Description

Steps to reproduce
1.open the full calendar page (be it a demo version or anything else)
2. Switch on the Narrator on NVDA screen reader
3. Now navigate to Week button using tab key and invoke it now navigate in the table using tab key and enable narrator and now verify that screen reader is announcing row header in scan mode or in browse mode.

Actual result:

When we navigate inside the week table the screen reader is not announcing the row header (all day, 5am, 6am......) and row headers are also not associated with the events.

Expected result:

When we navigate inside the week table screen reader should announce row header (all day, 5am, 6am......) both in browse mode and scan mode and row headers should also associate with the events.

**Comment by acerix** [comment]
I believe this existing ticket captures this need:
https://github.com/fullcalendar/fullcalendar/issues/6637
↑
Please visit the above issue, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates.
